src/bunit.web/Extensions/InputFile/BUnitBrowserFile.cs

```diff
@@ -0,0 +1,19 @@
+﻿#if NET5_0_OR_GREATER
+using Microsoft.AspNetCore.Components.Forms;
+
+namespace Bunit;
+
+internal class BUnitBrowserFile : IBrowserFile
+{
+    public Stream OpenReadStream(long maxAllowedSize = 512000, CancellationToken cancellationToken = new())
```

Comment:
I assume default value in the maxAllowedSize is some standard somewhere.

```suggestion
    public Stream OpenReadStream(long maxAllowedSize = 512000, CancellationToken cancellationToken = default)
```

Prefer the default style of declaring a default value for cancellation tokens.